slides.md

```diff
@@ -1,6 +1,6 @@
 ---
 theme: slidev-theme-nearform
-layout: image-right
```

Comment:
If we still want an image for the intro slide, we just have to add another `image: url` entry to this section, and keep the previous `image-right` layout. It's just that I don't want to choose on behalf of the authors.